README.md

```diff
@@ -24,6 +29,8 @@ In WhlgPublicWebsite run `npm install`
 
 ### Minio
 
+If using docker minio will be configured automatically.
```

Comment:
Can we move all these steps that made redundant by Docker to a "Manual setup (optional)" or similar section? This line is easily missed in my opinion.